tools/walletextension/wallet_extension.go

```diff
@@ -478,3 +478,19 @@ func (w *WalletExtension) Version() string {
 func (w *WalletExtension) GetTenNodeHealthStatus() (bool, error) {
 	return w.tenClient.Health()
 }
+
+func (w *WalletExtension) GetMessage(encryptionToken string, formatsSlice []string) (string, error) {
```

Comment:
let's rename to "GenerateUserMessageToSign"

Reply:
renamed, good suggestion